lmnet/lmnet/post_processor.py

```diff
@@ -419,3 +419,74 @@ class Softmax(Processor):
     def __call__(self, outputs, **kwargs):
         results = _softmax(outputs)
         return dict({'outputs': results}, **kwargs)
+
+
+class FormatJoints(Processor):
```

Comment:
How about changing this class name to` GaussianHeatmapToJoints` or something like that?

I'm a little bit unconfortable with this class name, since it starts with "Format" (verb) but it's not consistent with other class name, say, `FormatYoloV2`. With the latter name, it is very obvious for me, I can easily understand that this post processor will format the output of YoloV2 network. On the other hand, what the `FormatJoints` means is not obvious for me. Actually, at first, I misunderstood that this class expects Joint information as input, and output something which could be computed by the joints.

Reply:
This name was trying to imitate `FormatYoloV2`.
`GaussianHeatmapToJoints` should be better here as you said. 👍